viral/grosmark_analysis.py

```
@@ -141,64 +146,72 @@ def get_place_cells(
-    # Probably delete cache logic once we're all sorted
     use_cache = True
-    cache_file = (
+
+    get_cache_path = lambda variable_name: (
         SERVER_PATH
         / "viral_caches"
         / "place_cells"
-        / f"{session.mouse_name}_{session.date}_rewarded_{rewarded}_{config}_shuffled_matrices.npy"
+        / variable_name
+        / f"{session.mouse_name}_{session.date}_rewarded_{rewarded}_{config}_{variable_name}_BOD_{bin_occupancy_divide}.npy"
     )
```

Ruff (0.14.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `get_cache_path` as a `def`

(E731)